.github/workflows/centos7/entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh -l
+set -x
+uname -a 
+cat /etc/issue
+yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+yum -y install  clang clang-devel clang-libs  llvm-devel llvm-static
+yum -y install  libcxx-devel cmake3 make cmake gcc gcc-c++ 
+yum -y install yum-plugin-copr
```

Comment:
I use only the official docker images from Fedora. It makes no sense to create and maintain own images.

Reply:
Hm... i think you missing my point: if we put install procedure into entry-point script it will be executed _each_time_container_run_. However if we put it into docker-file it will be run _only_once_ during container build. Obviously later will take much less resources.

Reply:
lets move all follow up on this into #98

Reply:
OK. See #98